test/models/vote_test.rb

```diff
@@ -0,0 +1,30 @@
+require "test_helper"
+
+describe Vote do
+  before do
+    @vote = votes(:vote)
+  end
+
+  describe 'validations' do
+    it "is valid with work and user" do
+      expect(@vote.valid?).must_equal true
+    end
+    it "is invalid if user has already voted for work" do
+      @error_vote = Vote.new(
```

Comment:
This is a solid start, but there's a couple other interesting cases that I would like to see here:

- A user can have votes for two different works
- A work can have votes from two different users